core-compose/src/main/java/com/squareup/workflow/ui/compose/ComposeViewFactory.kt

```diff
@@ -55,6 +55,13 @@ import kotlin.reflect.KClass
  *
  * val viewRegistry = ViewRegistry(FooBinding, …)
  * ```
+ *
+ * ## Nesting child renderings
+ *
+ * Views that act as containers (i.e. they delegate to the
```

Comment:
Is there a clearer term than "View" that we can use here, to reinforce that we're talking about Compose? e.g. View function?

Reply:
I think this whole paragraph reads pretty badly. Will reword and check back with you.